splunk-quarkus/src/main/java/com/redhat/console/notifications/splunkintegration/SplunkIntegration.java

```diff
@@ -208,20 +235,10 @@ private void configureHandler() throws Exception {
                 // (see https://camel.apache.org/components/latest/http-component.html#_which_http_method_will_be_used).
                 .setHeader(Exchange.HTTP_URI, simple("$simple{headers.metadata[url]}"))
                 .setHeader(Exchange.HTTP_PATH, constant("/services/collector/event"))
-                .choice()
-                .when(simple("${headers.metadata[url]} startsWith 'http://'"))
-                .to(http("dynamic")
-                        .httpMethod("POST")
-                        .advanced()
-                        .httpClientConfigurer(getClientConfigurer()))
-                .endChoice()
-                .otherwise()
                 .to(https("dynamic")
                         .httpMethod("POST")
                         .advanced()
                         .httpClientConfigurer(getClientConfigurer()))
-                .endChoice()
-                .end()
```

Comment:
Can these two be kept and moved to bottom? It should avoid future bugs.

Reply:
Keeping `.end()`. I can't keep `.endChoice` as `.choice` was removed in line 211.